test/built-ins/Temporal/PlainDateTime/prototype/toZonedDateTime/options-object.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindate.prototype.tozoneddatetime
+description: Empty object may be used as options
+features: [Temporal]
+---*/
+
+const dt = new Temporal.PlainDateTime(2019, 10, 29, 10, 46, 38, 271, 986, 102);
+const expected = "2019-10-29T10:46:38.271986102-03:00[America/Sao_Paulo]";
+
+assert.sameValue(
+  `${dt.toZonedDateTime("America/Sao_Paulo", {})}`,
```

Comment:
- Use "UTC" since "America/Sao_Paulo" will not be a valid time zone for non-Intl implementations
- Prefer not using `toString` to do the assertion, instead check the expected value of `.epochNanoseconds`.

(Both these comments apply to most tests in this PR. In some cases you might also need to check `.timeZone` and `.calendar`)

Reply:
Good point -- I've updated the tests to use UTC or, in some cases, the spring-ahead-fall-back setup in `temporalHelpers.js`. I've tweaked things so that we don't test against the stringified object, but compare the epoch nanoseconds. That said, I'm not sure where to test the time zone and calendar, though. Any advice on where that makes sense, and where not?

Reply:
I only saw one place where I'd recommend testing the time zone. I added a comment marking it in my latest review.